### PR TITLE
Make buildSetCookieHeaderFromBrowserCookie robust

### PR DESCRIPTION
**Pull request by szepeviktor**
Now you can clearly see what is going on in `buildSetCookieHeaderFromBrowserCookie`.

Review: https://github.com/crwlrsoft/crawler/blob/f0da585ee8798e49a837b02f741c398a9356259e/src/Loader/Http/Cookies/CookieJar.php#L124

### CODE REVIEW

**Comment by szepeviktor** [comment]
💥 Those true values!
```
!testcookie=foo123; Domain=localhost; Path=/; Secure=; HttpOnly=
```

**Comment by szepeviktor** [comment]
The computer is sad

```
Failed to load http://www.crwlr.software/packages/query-string/v1.0/getting-started:
 cURL error 7: Failed to connect to localhost port 8003 after 0 ms: Connection refused
```

File: src/Loader/Http/Cookies/CookieJar.php [hunk 1]
```diff
@@ -123,37 +123,42 @@ protected function getForDomainFromUrl(string|UriInterface|Url $url): ?string
...
+            'secure' => 'Secure',
+            'httpOnly' => 'HttpOnly',
+            'sameSite' => 'SameSite',
+        ];
+
+        $parts = [sprintf('%s=%s', $cookie->getName(), $cookie->getValue())];
+
+        foreach ($attributes as $name => $setCookieName) {
+            $setCookieValue = $cookie->offsetGet($name);
+            if (empty($setCookieValue)) {
```
Comment (szepeviktor) [inline src/Loader/Http/Cookies/CookieJar.php:140]: `empty` is one of the bad parts of PHP, it is too complex.
It handles: null, empty string, false.

**Comment by otsch** [comment]
> The computer is sad
> 
> ```
> Failed to load http://www.crwlr.software/packages/query-string/v1.0/getting-started:
>  cURL error 7: Failed to connect to localhost port 8003 after 0 ms: Connection refused
> ```

@szepeviktor Next time please a little more information what you're trying to say...what the comment is about. I guess it's from running tests locally on your machine? 🤔 Not sure what's going on there...they're working on my machine as well as in gh actions.

**Comment by szepeviktor** [comment]
It was here on GitHub. One of the workflows failed.

image

https://github.com/crwlrsoft/crawler/actions/runs/11473029647/job/31926578720

**Pull request merged**